src/scripts/constants.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+_DEFAULT_TEST_COUNT = 100 # The Default value of --test-count option
+_DEFAULT_SEED = 10000 # The default value of random seed
+_TEST_CASE_IN_A_FILE = 700 # The number of test cases before start new file
+_OPERAND_COUNT_MIN = 2 # The --operand-count option's minimum value
+_OPERAND_COUNT_MAX = 11 # The --operand-count option's maximum value
+_MAX = (1 << 53) - 1 # The max value of random number
+_MIN = -1 * _MAX  # The min value of random number
```

Comment:
Please add a reference to `Number.{MAX, MIN}_SAFE_INTEGER` if these constants represent them.